BDT_application/step1_Samplepreparation/slim_mc.py

```diff
@@ -19,7 +19,7 @@ def Slim_module(filein,nin,mass_flag, use_fortraining, era):
 
   path = str(inputFile_path[era])
 
-  filters      = "ttc_jets && ttc_l1_pt > 30 && ttc_met > 30 && ttc_mll > 20 && ttc_drll > 0.3"
+  filters      = "ttc_jets && ttc_l1_pt > 30 && ttc_met > 30 && ttc_mll > 20 && ttc_drll > 0.3 && nHad_tau==0 && ttc_2P0F"
```

Comment:
wasn't there a second cut that Meng also sent to the zoom chat today?

Reply:
Hi @efeyazgan This is for prompt backgroud that yesterday we found missing. Just commited today.

the cut what Meng said in the meeting its there already for non propmpt case its here:
https://github.com/ExtraYukawa/Script_ForMVA/blob/8eacd12c639e811f333a94f47b07c6ee5d15a653/BDT_application/step1_Samplepreparation/slim_fakelep_mc.py#L21-L37